src/components/facets/FacetReducers.ts

```diff
@@ -0,0 +1,94 @@
+import { IFacetActionPayload, FacetActions } from './FacetActions';
+import { IReduxAction } from '../../utils/ReduxUtils';
+import { IFacet } from './Facet';
+import * as _ from 'underscore';
+
+export interface IFacetState {
+  facet: string;
+  opened: boolean;
+  selected: IFacet[];
+}
+
+export const facetInitialState: IFacetState = { facet: undefined, opened: false, selected: [] };
+export const facetsInitialState: IFacetState[] = [];
+
+export const facet = (state: IFacetState = facetInitialState, action: IReduxAction<IFacetActionPayload>): IFacetState => {
+  switch (action.type) {
+    case FacetActions.toggleMoreFacetRows:
```

Comment:
+1 

I know it's just a question of preferences, but I like the idea of having all components implemented the same way.